closure/goog/useragent/useragent.js

```diff
@@ -207,8 +194,12 @@ goog.userAgent.SAFARI = goog.userAgent.WEBKIT;
  * @private
  */
 goog.userAgent.determinePlatform_ = function() {
-  var navigator = goog.userAgent.getNavigator();
-  return navigator && navigator.platform || '';
+  const nav = goog.userAgent.getNavigator();
+  if (nav === null || !('platform' in nav)) {
```

Comment:
If you want to check against null use `goog.isNull`. Also `'platform' in nav` introduces a bug since the key might exist even though the value is undefined. Also you can't use in on null and nav might be null.